src/Playhead.jsx

```diff
@@ -2,28 +2,31 @@ import React from 'react';
 
 
 export default class Playhead extends React.Component {
+    constructor(props) {
+        super(props);
+        this.trackWidth = 915;
```

Comment:
This number comes from the design - the track width is the total width of the tool (970) - https://github.com/ccnmtl/mediathread/blob/master/media/juxtapose/css/juxtapose.css#L19 minus the width of the track icon (55) https://github.com/ccnmtl/mediathread/blob/master/media/juxtapose/css/juxtapose.css#L80

It won't change unless we change the design, in which case it would be obvious that this value will need to change since the playhead would behave incorrectly.